imports/ui/lists/recentAgents.coffee

```diff
@@ -34,9 +49,59 @@ Template.recentAgents.onCreated ->
         if @recentAgents.find(articleId: article._id).count() > 5
           @articles.update(article._id, { $set: { collapsed: true } })
 
+
+Template.recentAgents.onRendered ->
+  Meteor.startup =>
```

Comment:
Must've been a left-over from my previous implementation.
I believe I was waiting there for all the templates in the viewport to render.